src/components/headers/headerDao.tsx

```diff
@@ -58,7 +61,8 @@ export const HeaderDao: React.FC<HeaderDaoProps> = ({
     favorited = false,
     links = [],
     translation,
-    copiedOnClick,
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    onCopy = () => {},
```

Comment:
You can keep the `onCopy` property as undefined and when you need to call it you can simply do it with:
```
onClick={() => onCopy?.(daoEnsName)}
```

```suggestion
    onCopy,
```